components/tools/OmeroWeb/omeroweb/feedback/views.py

```diff
@@ -180,7 +167,11 @@ def handler500(request):
     if request.is_ajax():
         return HttpResponseServerError(error500)
 
-    return custom_server_error(request, error500)
+    form = ErrorForm(initial={'error': error500})
```

Comment:
I've filed https://trello.com/c/ra7UKuEJ/50-resurrect-5-1-web-method -- since I missed it in the first go, let's handle via a separate PR. Merging this.